Compiler/Parser/parser.py

```diff
@@ -612,15 +635,29 @@ def Compile(code: list) -> str:
                         code_string += node.visit() + ";\n"
                     else:
                         # Syntax is voilated
-                        print(
+                        error_list.append(
                             SyntaxError(
                                 line_no,
                                 check_Expr(line)[1]
                             )
                         )
         line_no += 1
+    '''
+    Even if a single error is there in a code whole converted C++ code will be deformed.
+    '''
+    if len(error_list) > 0:
```

Comment:
first check is any error in code than print all errors than exit() 
simple  because of this modification csq compiler not create cpp and not compile it